docs/_docs/auth.md

```diff
@@ -100,17 +110,27 @@ You can include any field your authentication test API call returns in the conne
 
 ![Zapier connection label with code](https://cdn.zapier.com/storage/photos/e9bf3fdaabe078e43b38a402106c1456.png)
 
-You can customize your connection label further with JavaScript code, if needed. As before, Zapier will always include your app's name and integration version number first, followed by any text returned by your connection label code.
+You can customize your connection label further with JavaScript code if needed. Zapier will always include your app's name first, followed by any text returned by your connection label code.
 
-Custom code for connection labels are best used for:
+Custom code for connection labels is best used for:
 
 - Using code to manipulate data from an auth or test call before using it in a connection label, such as to format a number or date
-- Logging additional data to the authentication log
+- Logging additional data
 - Making a new API call to access data to use in the connection label
 
-Click the _Edit Code_ button to switch to code mode. If the code mode is visible, Zapier will only use the code to create your connection label, and will ignore any existing text in the connection label field. If you wish to switch back to the regular field, click the _Delete Code_ button to delete your custom connection label code and revert to using the text entered in the connection label field.
+Click the _Edit Code_ button to switch to [Code Mode](./faq#how-does-code-mode-work). If the Code Mode is active, Zapier will only use the code to create your connection label, and will ignore any text that was in the _Connection Label_ field in the regular string mode. 
+
+If you want to switch back to the regular field, click the _Delete Code_ button to delete your custom connection label code and revert to using the text entered in the _Connection Label_ field.
+
+When writing code for the connection label, fields are not pulled up to the top level - you must use `bundle.authData` for auth input fields, or `bundle.inputData` for fields returned from the test API request. For example, your code might look like this:
+
+```
+return `bundle.authData.username`;
+```
+
+This is equivalent to using {% raw %}`{{username}}`{% endraw %} in the regular string mode.
```

Comment:
I see you use `{% raw %}` in many places. Do we need those? (genuine question)

Reply:
Everything in the docs that uses double curly braces that need to render puts those raw bits around them, so I'm following that. If we really don't need that, it would be a big change throughout - it seems like we could. explore that separately.